### DISCUSSION TITLE
Catpuccin Theme for glow

### DISCUSSION THREAD

**Discussion by jotihojr** [charmbracelet/glow#591] [General]
@bashbunni I'm not sure if I misunderstood your site https://www.bashbunni.dev... But is that glow output there also themed with catppuccin? Or which style did you user for that? None of the themes `-s <style>` I've tried seem to look like that.

Thanks,

**Comment by bashbunni** [comment]
Oh, just the site itself uses the Catppuccin colour palette! The glow output on my site should be just the default dark theme for glow
